### PR TITLE
Add friend list

### PR DESCRIPTION
**Pull request by arringtonh**

### CODE REVIEW

File: proj/src/main/webapp/chat.js [hunk 1]
```diff
@@ -864,6 +868,49 @@ function addTagsToDom(uid) {
...
+        } else {
+            const url = snap.val().photo || DEFAULT_PFP;
+            pfpRef = firebase.storage().refFromURL(url);
+            pfpRef.getDownloadURL().then(function(src) {
+                friend.querySelector('#pfp').src = src;
+                sessionStorage[uid+" pfp"] = src;
+            })
+        }
+
+        loadProfileOfSender(friend, uid) // to go profile on click
```
Comment (aokeke) [inline proj/src/main/webapp/chat.js]: `// to go profile on click`->`// to go to profile on click`

**Review by aokeke — APPROVED** [review]
LGTM

**Review by aokeke — APPROVED** [review]
LGTM

**Review by leeannachen — APPROVED** [review]
lgtm

**Pull request merged**